packages/edit-site/src/components/page-pages/index.js

```diff
@@ -136,7 +139,8 @@ export default function PagePages() {
 				header: 'Status',
 				id: 'status',
 				cell: ( props ) =>
-					postStatuses[ props.row.original.status ]?.name,
+					postStatuses[ props.row.original.status ] ??
```

Comment:
I thought I'd also update the fallback we show when the post statuses retrieval goes wrong (statuses are unavailable or just take a while to load): instead of leaving them blank, we can use the already available `status` property of the page.